src/components/workflowEditor/nodes/WebhookNode.tsx

```diff
@@ -24,6 +26,29 @@ export function WebhookNode({ x, y, width, height, id, to, configDiffView }: Web
                         <Webhook className="icon-dim-20" />
                     </div>
                 </ConditionalWrap>
+
+                {toggleCDMenu && (
+                    <button className="workflow-node__add-cd-btn">
+                        <Tippy
+                            className="default-tt"
+                            arrow={false}
+                            placement="top"
+                            content={
+                                <span style={{ display: 'block', width: '145px' }}> Add deployment pipeline </span>
+                            }
+                        >
+                            <Add
+                                className="icon-dim-18 fcb-5"
+                                onClick={(event: any) => {
```

Comment:
Can we extract this to a separate function?

Reply:
done